lua/wire/server/wirelib.lua

```diff
@@ -578,28 +577,18 @@ function WireLib.Link_Start(idx, ent, pos, iname, material, color, width)
 
 	local input = ent.Inputs[iname]
```

Comment:
@AbigailBuccaneer this is a local variable, your argument is invalid.
luacheck is going to complain about shadowing a global, too.